models/setup.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+from setuptools import setup, find_packages
+
+setup(
+    name="flood_ml",
```

Comment:
Do you intend to have one package for each model? Or one package containing both models? Right now, we still have one package named flood_model. You'd either want
```
models/
  - setup.py
  - models/
    -  __init__.py
    - flood_model/
    - wrf_mode/
```
for one package containing both models or
```
flood_model/
  - setup.py
  - flood_model/
    -  __init__.py
wrf_model/
  - setup.py
  - wrf_model/
    -  __init__.py 
```
for one package for each model.

I don't feel stro

Reply:
Sorry, what's the difference? I'm not really familiar with this packaging structure, so I don't understand how this might impact workflows down the line.

Reply:
The first 'one package containing both models' option would result in imports like
`from models.flood_model import ...`
The second 'one package for each model' option would be
`from flood_model import ...`

The folder sitting beside the `setup.py` will become the namespace for your package (either 'model' or 'flood_model' in the imports above.)
On that note, would probably want a namespace more distinct that 'model.' 'usl_model' would match what we're doing with 'usl_lib' and 'usl_pipeline' elsewhere in here.

Reply:
Down the line, it will determine things like whether we need to maintain two docker images when we pull things into vertex ai. I'd just recommend one package containing both models ala

```
models/
  - setup.py
  - requirements.txt
  - usl_models/
    -  __init__.py
    - flood_model/
    - wrf_mode/
```

for imports like
`from usl_models.flood_model import ...`

but we only have to maintain one requirements.txt file and one docker image for the two models.

Reply:
Yeah, I had figured that a single package would allow us to bundle some common things like the requirements file. Is it not possible to have a structure like this?

```
usl_models/
  - setup.py
  - requirements.txt
  - flood_model/
    - __init__.py
  - wrf_model/
    - __init__.py
  - tests/
```

Reply:
That creates two namespaces, `flood_model` and `wrf_model`. It's unusual for one package to own two different name spaces. I think what you have would work, it would just be unusual. The more common thing would be to stick flood_model & wrf_model under a single namespace and export that single namespace in your package.

Reply:
Updated the structure so it's a single `usl_models` package, based on your suggestion.